### PR TITLE
Copy change about the project

### LINKED ISSUE #110: Copy Edit: about the project page
## Change opening sentence to two sentences with slight word order edit.

In 2017, The Haven Wolverhampton worked with Neontribe and women who had been through domestic abuse. We co-designed a web app to encourage other women in that situation to seek support and advice.

## Acceptance Criteria
- [x] On the About the project page (info-to-help/about), the first 2 sentences are "In 2017, The Haven Wolverhampton, worked with Neontribe and women who had been through domestic abuse. We co-designed a web app to encourage other women in that situation to seek support and advice. "

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Fixes #110 

## Description

- Re-wording of first two lines

🐧

### CODE REVIEW

File: src/Info.elm [hunk 1]
```diff
@@ -123,8 +123,8 @@ info =
       , slug = "about"
       , icon = "question-circle-o"
       , infoText =
-            [ """In 2017, The Haven, Wolverhampton, Neontribe and women who had been through domestic abuse co-designed a web app to encourage other women in
-            that situation to seek support and advice. We want to find out if the structure of that app could be used by different organisations. We believe
+            [ """In 2017, The Haven, Wolverhampton worked with Neontribe and women who had been through domestic abuse. We co-designed a web app to encourage other women in that situation to seek support and advice.
```
Comment (katjam) [inline src/Info.elm]: The Haven Wolverhampton is all one thing. No comma on their own website.

**Review by katjam — COMMENTED** [review]
I think take the comma out from The Haven, Wolverhampton... but again check with @katquatermass

**Comment by katjam** [comment]
image

File: src/Info.elm (hunk 1 above)
Comment (HedgehogHolly) [reply src/Info.elm]: Should it be The Haven Wolverhampton,  with a comma or without?

**Review by katjam — APPROVED** [review]
Perfect. :lollipop:

File: src/Info.elm (hunk 1 above)
Comment (katjam) [reply src/Info.elm]: I think `The Haven Wolverhampton worked with Neontribe...`

**Pull request merged**